modules/toolbar.js

```diff
@@ -107,7 +107,7 @@ class Toolbar extends Module {
           option.selected = true;
         }
       } if (input.value) {
-        input.classList.toggle('ql-active', input.value == formats[format]);  // Intentional ==
+        input.classList.toggle('ql-active', input.value === formats[format]);
```

Comment:
I believe the general case of <button value="1"> gets broken by this === change.